backend/de.metas.adempiere.adempiere/base/src/main/java-legacy/org/compiere/model/MakeUniqueLocationNameCommand.java

```diff
@@ -174,6 +177,17 @@ private String truncateAndMakeUnique(@NonNull final String name)
 
 		return nameUnique;
 	}
-}
 
+	public static List<String> getOtherLocationNames(
+			final int bpartnerId,
+			final int bpartnerLocationIdToExclude)
+	{
+		return Services.get(IQueryBL.class)
```

Comment:
pls introduce queryBL field

Reply:
and make this method not static.... or decide to move it from here if not needed here